firebase-config/firebase-config.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
     implementation 'com.google.firebase:firebase-measurement-connector:18.0.0'
     implementation 'com.google.android.gms:play-services-tasks:17.0.2'
 
+    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
+    implementation 'com.squareup.okhttp3:okhttp-sse:4.9.0'
```

Comment:
Any concerns with this library being marked as experimental in https://github.com/square/okhttp/tree/master/okhttp-sse

Reply:
I was planning on using in the mean time. We could implement our own SSE functionality without much difficulty I would presume since it is pretty straightforward. But like in the previous comment it may be better for us to just use HTTP 1.1 with chunked encoding.